<ui version="4.0" >
 <class>hallSetup</class>
 <widget class="QDialog" name="hallSetup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Setup</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLang" >
       <property name="text" >
        <string>Language:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboLang" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="labelFont" >
       <property name="text" >
        <string>select font:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbFont" >
       <property name="text" >
        <string>font...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbMultiLogin" >
     <property name="text" >
      <string>Allow to login with multiplayer IDs</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbSound" >
     <property name="text" >
      <string>Playing sound</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Chat Options</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Min sentences:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMinSentences" >
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>Max sentences:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMaxSentences" >
          <property name="maximum" >
           <number>1000</number>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="singleStep" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>max size of image to download in chat:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinMaxSize" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>0</hsizetype>
            <vsizetype>0</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="specialValueText" >
           <string>No Limit</string>
          </property>
          <property name="suffix" >
           <string>K</string>
          </property>
          <property name="maximum" >
           <number>2000</number>
          </property>
          <property name="singleStep" >
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbSaveChat" >
        <property name="text" >
         <string>Save friends' chat record</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbPopupChat" >
        <property name="text" >
         <string>Popup chat window when receives message</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbProxy" >
     <property name="title" >
      <string>Use Proxy</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>3</number>
      </property>
      <property name="spacing" >
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="labelProxy" >
          <property name="text" >
           <string>Proxy Type:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QComboBox" name="comboProxyType" >
            <item>
             <property name="text" >
              <string>Socks5 Proxy</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Socks4 Proxy</string>
             </property>
            </item>
            <item>
             <property name="text" >
              <string>Http Proxy</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="labelServer" >
          <property name="text" >
           <string>Server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leServer" />
        </item>
        <item>
         <widget class="QLabel" name="labelPort" >
          <property name="text" >
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePort" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUsername" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2_2" >
          <property name="text" >
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePassword" >
          <property name="echoMode" >
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>16</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
